react/components/EXPERIMENTAL_Table/hooks/useTableLineActions.tsx

```diff
@@ -0,0 +1,52 @@
+import LineAction, { LineActionObject } from '../LineActions'
+import React, { useMemo } from 'react';
+
+const NO_TITLE_COLUMN = ' '
+const LINE_ACTION_ID = 'lineAction'
+
+const useTableLineActions = ({
+  items,
+  columns,
+  lineActions
+}: hookInput): hookReturn => {
+  
+  const withLineActionItems = useMemo<Array<Object>>(() => {
```

Comment:
```suggestion
  const itemsWithLineActions = useMemo<Array<Object>>(() => {
```
I think that is a better naming 🤔